tests/chainer_tests/distributions_tests/test_kldivergence.py

```diff
@@ -74,5 +82,16 @@ def test_normal_normal_gpu(self):
         dist2 = self.make_normal_dist(True)
         self.check_kl(dist1, dist2)
 
+    def test_normal_normal_log_var_cpu(self):
+        dist1 = self.make_normal_dist_log_var()
+        dist2 = self.make_normal_dist_log_var()
+        self.check_kl(dist1, dist2)
+
+    @attr.gpu
+    def test_normal_normal__log_var_gpu(self):
+        dist1 = self.make_normal_dist_log_var(True)
+        dist2 = self.make_normal_dist_log_var(True)
+        self.check_kl(dist1, dist2)
+
```

Comment:
Could you add KL tests with mixed argument sets?
To do so, it'll be easier if `make_normal_dist_log_var` is `make_normal_dist` with a new argument (not a new method).